en/flight_controller/pixhawk_series.md

```diff
@@ -87,8 +87,14 @@ At very high level, the main differences are:
   More Redundancy.
   More RAM.
   More CAN buses.
-  Much more configurable & customizable
-  .([Pixhawk 5X](../flight_controller/pixhawk5x.md), Skynode)
+  Much more configurable & customizable.
+  ([Pixhawk 5X](../flight_controller/pixhawk5x.md), Skynode)
+- **FMUv6C:** 
+  ([Holybro Pixhawk 6C Mini](../flight_controller/pixhawk6c_mini.md), [Holybro Pixhawk 6C](../flight_controller/pixhawk6c.md))
+- **FMUv6X:** 
+  ([CUAV Pixhawk V6X](../flight_controller/cuav_pixhawk_v6x.md),[Holybro Pixhawk 6X](../flight_controller/pixhawk6x.md))
+- **FMUv6X-RT:**
+  ([Holybro Pixhawk 6X-RT](../flight_controller/pixhawk6x-rt.md))
```

Comment:
I noticed these boards are missing a "relative overview". While we're here, can you suggest some text?

Reply:
Good one hard to say to be honest, V6X/V6X-RT has the same PAB as V5X.
I think but not sure that both V6X/V6X-RT support CAN FD instead of CAN.
Faster MCU core (480Mhz V6X, 1Ghz V6X-RT) More RAM (1MB on V6X, 2MB on V6X-RT), V6X-RT has more flash compared to V6X/V5X 64Mb instead of 2Mb.

Reply:
Thanks. I added this. Will try get more info for the others.